core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualOutputWriter.scala

```diff
@@ -466,6 +467,7 @@ object QualOutputWriter {
   val RECOMMENDED_WORKER_NODE_TYPE = "Recommended Worker Node Type"
   val DRIVER_NODE_TYPE = "Driver Node Type"
   val TOTAL_CORE_SEC = "Total Core Seconds"
+  val IS_PHOTON = "Photon App"
```

Comment:
Is it part of the requirements to add a binary flag for "Photon App" in the output files?
If not, then this should be part of the App properties Map[String, String]. For example, a user running on dataproc, the columns will not be useful.

Reply:
Removed the binary flag as we can detect this information from `spark_properties.csv`